packages/snap-preact/components/src/utilities/mergeProps.ts

```diff
@@ -92,6 +92,11 @@ export function mergeProps<GenericComponentProps = ComponentProps>(
 			},
 			treePath,
 		};
+
+		if ((props as any).lang) {
+			(mergedProps as any).lang = (props as any).lang;
+		}
```

Comment:
Types with `props as ComponentProps` above, should work here as well perhaps?

Reply:
unfortunately lang doesn't exist on ComponentProps.. not every component has a lang, so that doesn't really work here.

Reply:
Riiight.